scripts/test/integration.sh

```diff
@@ -3,6 +3,8 @@
 set -x
 set -e
 
+export SOURCE_RENDERING_BEHAVIOR=all
```

Comment:
Do we need to export it to workflow config too https://github.com/astronomer/astronomer-cosmos/blob/main/.github/workflows/test.yml#L147

Reply:
Just added them to
- integration tests
- 1.5.4 tests
- integration expensive

Let's wait for the results of the CI...